docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -260,49 +253,43 @@ cd synthetics/examples/todos/ &&\
 will be run.
 
 [source,sh]
 ----
-# Run tests on the current directory. Please note the dot `.` which indicates 
-# that we want to run tests on the current directory
+# Run tests on the current directory. The dot `.` indicates
+# that it should run all tests in the current directory.
 npx @elastic/synthetics .
 ----
 
-Once you have your tests up and running, follow the steps in the <<synthetics-quickstart,quickstart guide>>
-to integrate with the provided Docker project template.
-
-// Results. . .
-// [source,sh]
-// ----
-// Journey: basic addition and completion of single task
-//    ✓  Step: 'go to app' succeeded (150 ms)
-//    ✓  Step: 'add task Dont put salt in your eyes' succeeded (79 ms)
-//    ✓  Step: 'check that task list has exactly 1 elements' succeeded (7 ms)
-//    ✓  Step: 'check for task 'Don't put salt in your eyes' in list' succeeded (50 ms)
-//    ✓  Step: 'destroy task 'Don't put salt in your eyes'' succeeded (75 ms)
-//    ✓  Step: 'check that task list has exactly 0 elements' succeeded (2 ms)
-
-// Journey: adding and removing a few tasks
-//    ✓  Step: 'go to app' succeeded (125 ms)
-//    ✓  Step: 'add task Task 1' succeeded (40 ms)
-//    ✓  Step: 'add task Task 2' succeeded (49 ms)
-//    ✓  Step: 'add task Task 3' succeeded (21 ms)
-//    ✓  Step: 'check that task list has exactly 3 elements' succeeded (8 ms)
-//    ✓  Step: 'destroy task 'Task 2'' succeeded (94 ms)
-//    ✓  Step: 'check that task list has exactly 2 elements' succeeded (5 ms)
-//    ✓  Step: 'add task Task 4' succeeded (36 ms)
-//    ✓  Step: 'check that task list has exactly 3 elements' succeeded (4 ms)
-
-// Journey: check that title is present
-//    ✓  Step: 'go to app' succeeded (139 ms)
-//    ✓  Step: 'check title is present' succeeded (27 ms)
-
-// Journey: check that input placeholder is correct
-//    ✓  Step: 'go to app' succeeded (121 ms)
-//    ✓  Step: 'check title is present' succeeded (18 ms)
-
-//  19 passed (2983 ms)
-// ----
+After testing locally, you can run the test suite using the
+Elastic Synthetics integration configuration or `heartbeat.yml`.
+
+* **{agent} and {fleet}**: Add a test suite when configuring the Elastic Synthetics integration.
+Use Monitor Type "Browser" and use the Source Type "Zip URL" pointing to a zip file containing the test project.
+See <<synthetics-quickstart-fleet>> for details.
+* **{heartbeat}**: Copy and paste the path to your zip file into `heartbeat.yml`.
+You can jump to <<synthetics-quickstart-step-two>> if you're using the provided Docker
+project template or spin up {heartbeat} yourself.
+
+TIP: If you're managing journeys in GitHub, you can automatically update your monitors as you deploy new versions
+by setting up a synthetic integration that will download and run journeys from your repository's ZIP file.
+When setting up your synthetic integration, enter both the repository's ZIP URL and the relative path for your
+journeys within the repository's ZIP file.
```

Comment:
This is borrowed from the blog post. Should we make it more generic so it's more relevant to users who are not working in GitHub? Or do you think using GitHub as an example helps illustrate the approach?